frontend/src/pages/ApartmentPage.tsx

```diff
@@ -115,6 +120,12 @@ const ApartmentPage = (): ReactElement => {
     return subscribeLikes(setLikedReviews);
   }, []);
 
+  useEffect(() => {
+    get<CardData[]>(`/buildings/all/${apt?.landlordId}`, {
+      callback: setOtherproperties,
```

Comment:
`otherProperties` is data that the page should have before the loading bar goes away. Can you change this `useEffect` on line 113 so that `setLoaded(true)` happens if `otherProperties` exists?
<img width="829" alt="Screen Shot 2022-05-04 at 5 53 47 PM" src="https://user-images.githubusercontent.com/57203639/166832062-50a55382-e3ee-43fc-a823-9a3770b0c58a.png">